### DISCUSSION TITLE
ImportError: cannot import name 'SimpleConv' from 'torch_geometric.nn'

### DISCUSSION THREAD

**Discussion by atul1234anand** [pyg-team/pytorch_geometric#9500] [Q&A]
Hi I cant find the SimpleConv operation in Pytorch geometric. On trying to import it, I get the above error.

The import statement used is:
`from torch_geometric.nn import SimpleConv`

**Comment by akihironitta** [comment]
Did you see the import statement somewhere in the docs? The correct location of the class is:
```python
from torch_geometric.nn import SimpleConv
```

**Reply by atul1234anand** [reply]
My bad. That was the import statement. Correction made above.

**Reply by akihironitta** [reply]
Do you mean you still see the error? Which version of PyG are you using?
